docs/kubernetes/windows-details.md

```diff
@@ -0,0 +1,335 @@
+
+1. Create an azure disk storage class
+
+option#1: k8s agent pool is based on blob disk VM
+```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azuredisk/storageclass-azuredisk.yaml```
+
+option#2: k8s agent pool is based on managed disk VM
+```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azuredisk/storageclass-azuredisk-managed.yaml```
+
+2. make sure storageclass is created successfully
+```
+kubectl get storageclass/azuredisk -o wide
+```
+
+3. Create a pvc for azure disk
+```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azuredisk/pvc-azuredisk.yaml```
+
+4. Make sure pvc is created successfully
+```
+kubectl get pvc/pvc-azuredisk -o wide
+```
+
+5. Create a pod with azure disk pvc
+```kubectl apply -f https://raw.githubusercontent.com/JiangtianLi/Examples/master/windows/azuredisk/iis-azuredisk.yaml```
+
+6. Watch the status of pod until its `STATUS` is `Running`
+```
+watch kubectl get po/iis-azuredisk -o wide
+```
+
+7. Enter the pod container to validate
+```
+kubectl exec -it iis-azuredisk -- cmd
+```
+
+
+### Multiple containers in a POD
+
+Copy this yaml to a file, then deploy it with `kubectl apply -f <filename.yaml>`
+
+It will run 2 containers:
+
+- iis-container: This is a basic static web serber
+- servercore-container: This will run a script that changes the index page every 10 seconds
+
+Once deployed, you can load the web page and refresh it to see the contents changing.
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: two-containers
+  name: two-containers
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: two-containers
+      name: two-containers
+    spec:
+      volumes:
+      - name: shared-data
+        emptyDir: {}
+
+      containers:
+
+        - name: iis-container
+          image: microsoft/iis:windowsservercore-1803
+          volumeMounts:
+          - name: shared-data
+            mountPath: /wwwcache
+          command: 
+          - powershell.exe
+          - -command 
+          - "while ($true) { Start-Sleep -Seconds 10; Copy-Item -Path C:\\wwwcache\\iisstart.htm -Destination C:\\inetpub\\wwwroot\\iisstart.htm; }"
+
+        - name: servercore-container
+          image: microsoft/windowsservercore:1803
+          volumeMounts:
+          - name: shared-data
+            mountPath: /poddata
+          command: 
+          - powershell.exe
+          - -command 
+          - "$i=0; while ($true) { Start-Sleep -Seconds 10; $msg = 'Hello from the servercore container, count is {0}' -f $i; Set-Content -Path C:\\poddata\\iisstart.htm -Value $msg; $i++; }"
+
+      nodeSelector:
+        beta.kubernetes.io/os: windows
+```
+
+
+## Troubleshooting
+
+Windows support is still in active development with many changes each week. Read on for more info on known per-version issues and troubleshooting if you run into problems.

```

Comment:
I recommend putting "active development" in bold to ensure those reading these docs are fully aware.